recipes-graphics/westeros/westeros-soc-drm.bb

```diff
@@ -13,7 +13,7 @@ DEPENDS = "wayland virtual/egl glib-2.0 libdrm"
 PROVIDES = "westeros-soc"
 RPROVIDES_${PN} = "westeros-soc"
 
-CFLAGS_append = " -I${STAGING_INCDIR}/libdrm"
+CFLAGS_append = " -I${STAGING_INCDIR}/libdrm -DWESTEROS_GL_NO_PLANES"
```

Comment:
This will hold good for all DRM platforms (Hikey, DB410c & iMX8) except Rpi-VC4graphics case where this flag has to be removed.

Reply:
I have added a CFLAGS_remove specific to rpi - vc4graphics.